unit-tests/py/rspy/repo.py

```diff
@@ -52,3 +52,31 @@ def pretty_fw_version( fw_version_as_string ):
     :return: a version with leading zeros removed, so as to be a little easier to read
     """
     return '.'.join( [str(int(c)) for c in fw_version_as_string.split( '.' )] )
+
+
+def find_built_exe( source, name ):
+    """
+    Find an executable that was built in the repo
+
+    :param source: The location of the exe's project, e.g. tools/convert
+    :param name: The name of the exe, without any platform-specific extensions like .exe
+    :return: The full path, or None, of the exe
+    """
+    exe = None
+    if platform.system() == 'Linux':
+        global build
+        exe = os.path.join( build, source, name )
+        if not os.path.isfile( exe ):
+            return None
+    else:
+        # In Windows, the name will be without extension and we need to find it somewhere
+        # in the path
```

Comment:
Why can't we search the build directory? Isn't that where the executables go?

Reply:
The idea with this PR is to add functionality to Linux, without touching Windows (LibCI will have different needs from running run-unit-tests manually, and I didn't want to couple two things all at once).
We can enhance the Windows capability as needed later -- but for now, this existing method worked.

Reply:
And executables in linux don't go into the build directory?